src/cdk/constructs/CloudFrontDistribution.ts

```diff
@@ -82,6 +98,13 @@ export class CloudFrontDistribution extends Construct {
           ],
           cachePolicy: splitCachePolicy
         },
+        '/_next/image': {
```

Comment:
_next/image* ?

Reply:
it won't have any path after `_next/image` since the rest is handled by query, but updated rule for now just avoid any collisions to make sure that it will point to next server